### PR TITLE
Added ArduinArm!

### PR DESCRIPTION
**Pull request by Skumm00**
## Describe your project in 1-3 sentences

I am building a Bluetooth-controlled Robot arm using Arduino.

## Hack Club Slack handle

Skand Bajpai

## Checklist before submitting the proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by Skumm00** [comment]
My Slack Handle is Skand Bajpai.

File: skumm00-proj.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+name: "@Skumm00"
+project: "Robot Arm"
+---
+
+#  Project Name - ArduinArm
+
+##  Summary
+
+My project is going to be a custom-controlled, Arduino-powered, robot arm.
```
Comment (exu3) [inline skumm00-proj.md:10]: Could you please elaborate a bit more here?

Some prompts you may wish to answer in this section:
What excites you about building this project? Have you built similar hardware projects before? Why do you want to build this project?

File: skumm00-proj.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+##  Plan
+
+I will be using a pre-existing set of 3d models for the overall structure of the robot arm. For the first 3 axes, the waist, the shoulder, and the elbow,
```
Comment (exu3) [inline skumm00-proj.md:14]: Could you link the 3D models here?

File: skumm00-proj.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Inventor in order to create the app to communicate with the robot.
+
+ARM MODEL: https://thangs.com/designer/m/3d-model/38899
+
+##  Budget
+
+| Product  | Supplier/Link  | Cost  |
+| --------------- | ------------------------------------- | ------ |
+| 3D printer  | [Amazon](https://www.amazon.com/Voxelab-Structure-Certified-Removable-8-66x8-66x9-84in/dp/B09BNG5884/ref=sr_1_8?crid=51ZBDXLXFTJD&keywords=3d%2Bprinter&qid=1671945440&sprefix=3d%2Bprint%2Caps%2C148&sr=8-8&ufe=app_do%3Aamzn1.fos.f5122f16-c3e8-4386-bf32-63e904010ad0&th=1) | $159.99  |
+| Arduino UNO | [Amazon](https://www.amazon.com/Arduino-A000066-ARDUINO-UNO-R3/dp/B008GRTSV6/ref=as_li_ss_tl?keywords=arduino+uno&qid=1580073330&sr=8-3&linkCode=sl1&tag=howto045-20&linkId=2de9a9cd18cdb5d608fbd42ba75f1298&language=en_US)  | $28.50 |
```
Comment (exu3) [inline skumm00-proj.md:29]: Do you have a USB B cable you can use? It looks like the Arduino linked does not come with one.

File: skumm00-proj.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+that constantly checks for outside input via bluetooth to the arm - if there is any, the corresponding motor will move. I will likely be using MIT App 
+Inventor in order to create the app to communicate with the robot.
+
+ARM MODEL: https://thangs.com/designer/m/3d-model/38899
+
+##  Budget
+
+| Product  | Supplier/Link  | Cost  |
+| --------------- | ------------------------------------- | ------ |
+| 3D printer  | [Amazon](https://www.amazon.com/Voxelab-Structure-Certified-Removable-8-66x8-66x9-84in/dp/B09BNG5884/ref=sr_1_8?crid=51ZBDXLXFTJD&keywords=3d%2Bprinter&qid=1671945440&sprefix=3d%2Bprint%2Caps%2C148&sr=8-8&ufe=app_do%3Aamzn1.fos.f5122f16-c3e8-4386-bf32-63e904010ad0&th=1) | $159.99  |
```
Comment (exu3) [inline skumm00-proj.md:28]: What kind of filament are you planning to use?

File: skumm00-proj.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+| Bluetooth Module | [Amazon](https://www.amazon.com/HiLetgo-Wireless-Bluetooth-Transceiver-Arduino/dp/B071YJG8DR/ref=as_li_ss_tl?ie=UTF8&qid=1533394818&sr=8-3&keywords=HC-05&linkCode=sl1&tag=howto045-20&linkId=86cdebbcfc701f763369f68487f8bbc9)  | $10.39 |
+| Breadboard Kit | [Amazon](https://www.amazon.com/Smraza-Breadboard-Resistors-Mega2560-Raspberry/dp/B01HRR7EBG/ref=sr_1_6?crid=1CBX6I7ZBZRHJ&keywords=breadboard+kit+arduino&qid=1671949226&sprefix=breadboard+kit+arduino%2Caps%2C147&sr=8-6)  | $12.99 |
+| Micro Servo Motors | [Amazon](https://www.amazon.com/Micro-Servos-Helicopter-Airplane-Controls/dp/B07MLR1498?crid=GE0AQEOIKH0G&keywords=SG90&qid=1668339645&sprefix=9g%2Bservo%2Bmotor%2Caps%2C539&sr=8-8&th=1&linkCode=sl1&tag=howto045-20&linkId=6fdd2d64320cbe9133dc62e4ee9e75d5&language=en_US&ref_=as_li_ss_tl)  | $19.95 |)  | $10.57 |
+| Power Adapter | [Amazon](https://www.amazon.com/iMBAPrice-Adapter-Listed-Supply-5-Feet/dp/B00GUO5WUI/ref=as_li_ss_tl?ie=UTF8&qid=1533394229&sr=8-3&keywords=DC%2B5V%2Bpower&linkCode=sl1&tag=howto045-20&linkId=b241c68f60eb10e244ed6ea9f08c9acf&th=1)  | $7.99 |
+| Total  | 257.80 + $17.30 tax  | $275.10 |
```
Comment (exu3) [inline skumm00-proj.md:35]: You can only get up to $250 with the grant; you'll have to cover the rest on your own by adding money to your Hack Club Bank account. Would you be able to cover this?

**Review by exu3 — CHANGES_REQUESTED** [review]
Hello @Skumm00, your project proposal is looking great so far. Could you please rename the file to follow the `githubusername.md` format and address the comments below?

**Comment by Skumm00** [comment]
@exu3 how does it look now?

**Comment by Skumm00** [comment]
I've just found out I do have a USB-B cable at home, which is why it isn't included in the budget.

File: skumm00.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+name: "@Skumm00"
+project: "Robot Arm"
+---
+
+#  Project Name - ArduinArm
+
+##  Summary
+
+My project is going to be a custom-controlled, Arduino-powered, robot arm. I've used 3-d printers and worked with Arduino circuits at my middle school, so I believe
```
Comment (exu3) [inline skumm00.md:10]:
```suggestion
My project is going to be a custom-controlled, Arduino-powered, robot arm. I've used 3D printers and worked with Arduino circuits at my middle school, so I believe
```

File: skumm00.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+this project shadows my skills well while still being a challenging build. Plus, this project would be a great introduction into the field of mechatronics, which is
+the usage of automated engineering (like 3-d printers and CNCs) for industrial development, which is a field I would like to consider in the future.
```
Comment (exu3) [inline skumm00.md:12]:
```suggestion
the usage of automated engineering (like 3D printers and CNCs) for industrial development, which is a field I would like to consider in the future. 
```

**Review by exu3 — APPROVED** [review]
Nice work! The @hackclub/bank-ops team will follow up with next steps on receiving your grant through Hack Club Bank.

Looking forward to seeing what you ship!

**Review by MelSmith104 — COMMENTED** [review]
hey @Skumm00 ! i can't wait to see your robot arm come to life - this looks like such a cool project! im dm'ing you on slack right now so that we can chat about next steps!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**